src/Testcontainers.FirebirdSql/FirebirdSqlContainer.cs

```diff
@@ -0,0 +1,85 @@
+namespace Testcontainers.FirebirdSql;
+
+/// <inheritdoc cref="DockerContainer" />
+[PublicAPI]
+public class FirebirdSqlContainer : DockerContainer, IDatabaseContainer
+{
+    public const string TestQueryString = "select 1 from RDB$DATABASE;";
+
+    private readonly FirebirdSqlConfiguration _configuration;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="FirebirdSqlContainer" /> class.
+    /// </summary>
+    /// <param name="configuration">The container configuration.</param>
+    /// <param name="logger">The logger.</param>
+    public FirebirdSqlContainer(FirebirdSqlConfiguration configuration, ILogger logger) : base(configuration, logger)
+    {
+        this._configuration = configuration;
+    }
+
+    private string GetDatabaseName()
+    {
+        if (State == TestcontainersStates.Running && IsFirebird2_5Image)
+        {
+            var dbPath = _configuration.Environments.TryGetValue("DBPATH", out var path)
+                ? path
+                : "/firebird/data";
+            return $"{dbPath}/{_configuration.Database}";
+        }
```

Comment:
Do we really need to support an outdated (EOL) version?

Reply:
I appreciate you taking the time to look this over!
I actually have the use case where our customer is (stuck) using a firebird v2.5.9 database, and we want to test specifically against that EOL version.
My case is that
a) this is also part of the java testcontainers implementation that comes from the firebird repo, so the idea isn't too farfetched and,
b) if it becomes more hassle to support v2.5, support can be dropped in a future version, but devs may still use the older version to use whatever functionality it offers.

Reply:
I am not sure about this section. In our project it _only_ works with v2.5 without settings the "/firebird/data" extra in front of the database, when using EntityFramework to access the database. I will do some more evaluation, but this is quite awkward.